source/_posts/generating-pdf-in-ruby-on-rails.md

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
blog/2020/02/26/generating-pdf-in-ruby-on-rails/ :  this url is not correct url, it should be in text format with hyphen   Correct URL: http://yourdomain.com/categroy/subcategory/tageted-url

Reply:
This can be done by @rana-abhishek

Reply:
Done